library/core/core.qs

```diff
@@ -3,20 +3,32 @@
 
 namespace Microsoft.Quantum.Core {
     /// # Summary
-    /// Returns the number of elements in an array.
+    /// Returns the number of elements in the input array `a`.
     ///
     /// # Input
     /// ## a
     /// Input array.
     ///
     /// # Output
-    /// The total count of elements in an array.
+    /// The total number of elements in the input array `a`.
+    ///
+    /// # Example
+    /// ```qsharp
+    /// Message($"{ Length([0, 0, 0]) }"); // Prints 3
+    /// ```
     function Length<'T>(a : 'T[]) : Int {
         body intrinsic;
     }
 
     /// # Summary
-    /// Creates an array of given length with all elements equal to given value.
+    /// Creates an array of given `length` with all elements equal to given
+    /// `value`. `length` must be a non-negative integer.
+    ///
+    /// # Description
+    /// Use this function to create an array of length `length` where each
+    /// element is equal to `value`. This way of creating an array is preferred
+    /// over other methods if all elements of the array must be the same and
+    /// the length is known upfront.
```

Comment:
Is this really the preferred way? I would have thought `[One, size = 3]` syntax would be preferred.

Reply:
@swernli may have more on this, but I believe we want to deprecate [One, size=3] syntax eventually.

Reply:
Yeah, that was the main motivation behind introducing `Repeated` and putting it in the core. Hopefully we can discourage usage of `[One, size = 3]` enough to make it less painful in some future where it is removed.